tests/integration/test_unicode_mojibake_issue162.py

```
@@ -0,0 +1,127 @@
+        _ = execute_powershell(cmd)
+
+        # Read back
+        result = view(test_file)
+
+        print("\nEcho redirection - View result:")
+        print(result)
+
+        # Check for mojibake
+        if "â†" in result or "âœ…" in result:
+            print("\n❌ MOJIBAKE in echo redirection!")
+            raise AssertionError("Mojibake detected in echo redirection")
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)